### PR TITLE
Fixing bugs in CollectMultipleMetrics

### PR DESCRIPTION
**Pull request by kbergin**
Needs to notify a user if reference sequence is not specified but is needed. Also removes CollectGcBiasMetrics from the list of default programs. In response to #225.

### CODE REVIEW

File: src/java/picard/analysis/CollectMultipleMetrics.java [hunk 1]
```diff
@@ -184,7 +217,10 @@ public int doWork() {
 
         final List<SinglePassSamProgram> programs = new ArrayList<SinglePassSamProgram>();
         for (final ProgramInterface program : new HashSet<ProgramInterface>(programsToRun)) {
-            final SinglePassSamProgram instance = program.makeInstance(OUTPUT , INPUT, REFERENCE_SEQUENCE);
+            if (program.needsReferenceSequence() && REFERENCE_SEQUENCE==null) {
+                throw new PicardException("A program you are trying to run needs a Reference Sequence, please set REFERENCE_SEQUENCE in the command line");
```
Comment (nh13) [inline src/java/picard/analysis/CollectMultipleMetrics.java]: Instead of ```a program``` can you print the program's name?

Comment (kbergin) [inline src/java/picard/analysis/CollectMultipleMetrics.java]: I was hoping to be able to but couldn't figure it out right away. Let me look into it further.

Comment (yfarjoun) [inline src/java/picard/analysis/CollectMultipleMetrics.java]: won't program.toString() do it?

Comment (kbergin) [inline src/java/picard/analysis/CollectMultipleMetrics.java]: Yes it does, I figured that out finally and added a new commit where it does that a little while ago. Thanks Yossi!

**Comment by bradtaylor** [comment]
Looks good to me!

**Comment by kbergin** [comment]
Thank you!

**Pull request merged**